src/style/front-end/components/core-blocks.scss

```diff
@@ -227,9 +227,9 @@ ul.wp-block-latest-posts.wp-block-latest-posts__list{
       }
       a{
         color:$black;
-        font-family:$source-serif-pro;
+        font-family:$acumin-pro-semicondensed;
         font-size:1.5625rem;
-        line-height:2rem;
+        line-height:1.35rem;
```

Comment:
Can we change it to line-height:1.3? 1.35rem is a little too small for this font size.

Reply:
Fixed the line line height. The news block was using 1.35, so used that to be consistent.